src/ArmTemplates/Extractor/Models/ApiParameterProperty.cs

```diff
@@ -0,0 +1,20 @@
+﻿// --------------------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+// --------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Management.ApiManagement.ArmTemplates.Extractor.Models
+{
+    public class ApiParameterProperty
+    {
+        public string Oauth2Scope { get; private set; }
```

Comment:
No need for `private` keyword here, due to properties are assigned in constructor

Reply:
Same in `ServiceUrl` property

Reply:
```suggestion
        public string Oauth2Scope { get; set; }
```

Reply:
Actually you can delete **setter**, because you are not making an assignment to that ApiProperties. Or you do?